staging/src/kubevirt.io/api/core/v1/types.go

```diff
@@ -817,6 +817,8 @@ const (
 	// This label represents vendor of cpu model on the node
 	CPUModelVendorLabel = "cpu-vendor.node.kubevirt.io/"
 
+	ModelVirtio = "virtio"
```

Comment:
It seems that it is sometimes referred as `model`, sometime as `type` and sometimes `bus`.
How about just calling it `VirtIO`? Lets see what others think, this one is not a blocker for me.

Reply:
Yep, i meant that it is the x model for example disk model / interface model (maybe even bus model)

https://github.com/kubevirt/kubevirt/pull/8219/commits/d9502000ecb2236c7b025f9b7e49817dc211912d#diff-de97cdb6be4b3bd722d6e7e208a39934957167715eef17717bf4c7009cc40cf5R171

does it make more sense, or you still prefer `VirtIO` only ?